en/mozillavpn.xliff

```diff
@@ -0,0 +1,556 @@
+      <note annotates="source" from="developer">the current upload speed</note>
+    </trans-unit>
+    <trans-unit id="vpn.connectionInfo.close">
+      <source xml:space="preserve">Close</source>
+      <context-group purpose="location"><context context-type="linenumber">156</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/components/VPNConnectionStability.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.connectionStability.checkConnection">
+      <source xml:space="preserve">Check Connection</source>
+      <context-group purpose="location"><context context-type="linenumber">138</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/components/VPNControllerDevice.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.devices.myDevices">
+      <source xml:space="preserve">My devices</source>
+      <context-group purpose="location"><context context-type="linenumber">14</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewDevices.qml</context><context context-type="linenumber">42</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.devices.activeVsMaxDeviceCount">
+      <source xml:space="preserve">%1 of %2</source>
+      <context-group purpose="location"><context context-type="linenumber">37</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">46</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewDevices.qml</context><context context-type="linenumber">23</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewDevices.qml</context><context context-type="linenumber">32</context></context-group>
+      <note annotates="source" from="developer">Context: You have "x of y" (y is max) devices in your account.</note>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/components/VPNControllerServer.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.servers.selectLocation">
+      <source xml:space="preserve">Select location</source>
+      <context-group purpose="location"><context context-type="linenumber">15</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewServers.qml</context><context context-type="linenumber">16</context></context-group>
+      <note annotates="source" from="developer">Select the Location of the VPN server</note>
+    </trans-unit>
+    <trans-unit id="vpn.servers.currentLocation">
+      <source xml:space="preserve">current location - %1</source>
```

Comment:
Should this be an *em dash* or a colon, and start uppercase? Not that it's really blocking, since it's an accessibility label, but it would be probably nicer.

> Current location: %1